ui/pages/management.js

```diff
@@ -42,16 +43,27 @@ export default function Lifestyle() {
   
 
   function ManageServiceMesh () {
+    const sample=[0,1,2,3,4]
```

Comment:
@sudo-NithishKarthik , i think changes are needed here, in the master branch `filtered ops` are used to map the cards, i also didn't understand the implementation of it. Your help might be required here